easybuild/easyblocks/generic/binary.py

```diff
@@ -106,8 +107,10 @@ def install_step(self):
         else:
             cmd = ' '.join([self.cfg['preinstallopts'], self.cfg['install_cmd'], self.cfg['installopts']])
             self.log.info("Installing %s using command '%s'..." % (self.name, cmd))
-            run_cmd(cmd, log_all=True, simple=True)
-
+            if self.cfg['qa'] is None:
+                run_cmd(cmd, log_all=True, simple=True)
+            else:
+                run_cmd_qa(cmd, self.cfg['qa'], log_all=True, simple=True)
```

Comment:
you can define `qa` as a custom easyconfig parameter specific to `Binary`, via the `extra_options` method in this easyblock